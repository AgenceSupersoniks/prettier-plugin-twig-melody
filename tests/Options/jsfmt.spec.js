run_spec(__dirname, ["melody"], {
    twigAlwaysBreakObjects: false,
    twigPrintWidth: 120
});
