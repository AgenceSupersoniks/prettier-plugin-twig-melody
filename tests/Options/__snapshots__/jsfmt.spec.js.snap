// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alwaysBreakObjects.melody.twig 1`] = `
<section class="{{ {
    base: css.prices
} | classes }}">
</section>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<section class="{{ { base: css.prices } | classes }}"></section>

`;

exports[`printWidth.melody.twig 1`] = `
<span attr1="one" attr2="two" attr3="three" attr4="four" attr5="five" attr6="six" attr7="seven" attr8="abcd">Text</span>

<span attr1="one" attr2="two" attr3="three" attr4="four" attr5="five" attr6="six" attr7="seven" attr8="abcde">Text</span>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<span attr1="one" attr2="two" attr3="three" attr4="four" attr5="five" attr6="six" attr7="seven" attr8="abcd">Text</span>

<span attr1="one" attr2="two" attr3="three" attr4="four" attr5="five" attr6="six" attr7="seven" attr8="abcde">
    Text
</span>

`;
